proposals/NNNN-Add-new-SDL-System-Structure-using-Mediation-Application.md

```diff
@@ -0,0 +1,119 @@
+# Add new SDL system Structure using Mediation Application for Middle/Low-end Class Model of Powered Two Wheeler and low-cost vehicle models
+
+* Proposal: [SDL-NNNN](NNNN-Add-new-SDL-System-Structure-using-Mediation-Application.md)
+* Author: [Akihiro Miyazaki (Nexty)](https://github.com/Akihiro-Miyazaki)
+* Status: **Awaiting review**
+* Impacted Platforms: [ iOS / Java Suite / JavaScript Suite / HMI / Policy Server / SHAID / RPC / Protocol ]
+
+## Introduction
+
+This proposal is to construct new SDL system for a Head Unit(HU) with low performed ECU for Powered Two Wheeler (PTW) *1 middle/low-end class models and low-cost vehicle models.
+
+*1 Powered Two Wheeler (PTW) includes motorcycle/scooter/moped.
+
+
+## Motivation
+
+The current SDL system is mainly for four-wheeled vehicles and requires a high-performance/high-resolution system.(The binary size is 177MB when built with x86_64.) However, it is not possible to mount a high-performance/high-resolution system on a PTW and low-cost vehicles due to the limitation of space and price that it has. Therefore, support for PTW and low-cost vehicles in the current SDL system is not incorporated. The main purpose of this proposal is to construct new SDL system for a Head Unit(HU) with low performed ECU.
+
+
+## Proposed solution
+
+Firstly, PTW can be classified into the following classes by their meter display specifications.
+ - High-end class: display specifications and ECU performance are equivalent to that of a four-wheeled vehicle.
+ - Middle-end class: a TFT liquid-crystal display but has restrictions such as the area assigned to the SDL is small, and ECU performance is low.
+ - Low-end class: simple display specifications and low ECU performance.
+
+Furthermore, the display specifications of low-cost vehicle models are also equivalent to PTW middle/low-end class models. Therefore, this proposal are for PTW middle/low-end class models and low-cost vehicle models.
+
+In the current SDL system configuration, HU implements SDL Core and SDL HMI as follows.
+
+<b>Figure1.</b> Current SDL system configuration
+
+![Figure1_Current_SDL_system_configuration.png](../assets/proposals/NNNN-Add-new-SDL-System-Structure-using-Mediation-Application/Figure1_Current_SDL_system_configuration.png)
+
+However, in the PTW middle/low-end models and low-cost vehicle models, since HU can only implement the low performed ECU, it can not implement the large programs such as SDL Core. Therefore, in this proposal, the target is to add new SDL system which move the SDL Core to smartphone like follows.
```

Comment:
```suggestion
However, in the PTW middle/low-end models and low-cost vehicle models, since the HU can only implement the low performance ECU, it cannot implement the large programs such as SDL Core. Therefore, in this proposal, the target is to add a new SDL system which moves SDL Core to the smartphone as follows.
```